package.json

```diff
@@ -44,8 +44,10 @@
         "@babel/preset-react": "^7.0.0",
         "@dhis2/app-runtime": "1.1.0",
         "@dhis2/cli-style": "3.1.1",
+        "@dhis2/d2-i18n": "1.0.4",
```

Comment:
Yarn doesn't install peer dependencies, so we need to state it as a dev dependency to get local development with Storybook to work.